databases/relational-databases.md

```diff
@@ -167,6 +123,20 @@ Further learning: Object-Relational Mapping (& ORM libaries)
 - [JavaTPoint sql quiz](https://www.javatpoint.com/sql-quiz)
```

Comment:
Needs new ‘check for understanding ‘ with questions instead (see #700 ). also, check off this outline in 700 while you’re there, please.

You can move these quizzes to ### Challenge or ### Supplemental Resources.

Reply:
Base your questions on the ### Objectives.

Reply:
Sorry, but @alodahl  can you please explain what exactly I need to check in #700.

Reply:
I just meant you should follow the rules in #700 .